seed/static/seed/partials/inventory_map.html

```diff
@@ -0,0 +1,135 @@
+<div class="page_header_container" ng-cloak>
```

Comment:
I completely copied this from the inventory_list.html partial. I commented out components that I believe will not be necessary (going forward for filtering, etc). I'll go in later and remove them completely once filtering is added or if we end up pushing this to the `develop` branch.